src/compiler/checker.ts

```diff
@@ -7607,6 +7661,11 @@ namespace ts {
             return links.resolvedSignature;
         }
 
+        /**
+         * Performs typechecking on the given node, which is a call/new expression.

```

Comment:
It's not just type-checking (the first thing it does is a grammatical validity check).

"Syntactically and semantically checks a call or new expression" would be better.